packages/test-utils/package.json

```diff
@@ -1,16 +1,28 @@
 {
   "name": "@terra-ui-packages/test-utils",
-  "version": "0.0.2",
+  "version": "0.0.4",
```

Comment:
Any reason you are skipping 0.0.3?

Reply:
Oops, looks like I bumped the version in two commits.

Reply:
Updated in https://github.com/DataBiosphere/terra-ui/pull/4294/commits/746f35d586efdbd07c4f01f8241f34faf9cf698d.